extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js

```diff
@@ -10,13 +10,27 @@ require(['wikia.window', 'jquery', 'wikia.tracker'], function (window, $, tracke
 	};
 
 	$(function () {
-		$('.rwe-page-header--dropdown').hover(function () {
-			$(this).addClass('rwe-page-header--active');
+		$('.rwe-page-header-nav__element-dropdown').hover(function () {
+			$(this).addClass('rwe-page-header-nav__element--active');
+			$('.rwe-page-header-nav__dropdown[data-category=' +
+				$(this).children('.rwe-page-header-nav__link').data('tracking') + ']').show();
 		}, function () {
-			$(this).removeClass('rwe-page-header--active');
+			$(this).removeClass('rwe-page-header-nav__element--active');
+			$('.rwe-page-header-nav__dropdown[data-category=' +
+				$(this).children('.rwe-page-header-nav__link').data('tracking') + ']').hide();
 		});
 
-		$('.rwe-page-header-nav__link').on('click', function () {
+		$('.rwe-page-header-nav__dropdown').hover(function () {
+			$('*[data-tracking=' + $(this).data('category') + ']').parent('.rwe-page-header-nav__element')
+				.addClass('rwe-page-header-nav__element--active');
+			$(this).show();
+		}, function () {
+			$('*[data-tracking=' + $(this).data('category') + ']').parent('.rwe-page-header-nav__element')
+				.removeClass('rwe-page-header-nav__element--active');
+			$(this).hide();
+		});
+
+		$('.rwe-page-header-nav__link').on('click', function (e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "e". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)